lib/server.js

```diff
@@ -82,18 +82,15 @@ module.exports = class AlgoServer extends EventEmitter {
   setAlgoHost (aoHost) {
     this.clearAlgoHost()
     this.host = aoHost
+
     // bind new events to new algo host
     this.host.on('ao:start', this.onAOStart)
     this.host.on('ao:stop', this.onAOStop)
-    this.host.on('ws2:notification', this.onWSNotification)
-    this.host.on('ws2:auth:error', this.onWSAuthError)
```

Comment:
Seems like this was a breaking change but since we've merged the algo-host 9https://github.com/bitfinexcom/bfx-hf-algo-server/pull/11/files) update its probably best if we merge this at version 1.3 and not a major version inc